website/package.json

```diff
@@ -4,6 +4,8 @@
   "main": "index.html",
   "license": "GPLv3",
   "scripts": {
+    "start:mainnet": "CHAIN_ID=1 npm start",
+    "start:rinkeby": "CHAIN_ID=4 npm start",
```

Comment:
We have some plans for multi-network support using the same front-end, so I think this should either be managed by the client themselves on the front-end somehow (possibly discovery) or by the API, if possible. (#14)

Reply:
Yes; I'm not very sure it's possible right now with `use-wallet` for doing this; might get a bit contrived for now, as we need to set a provider with a chain ID. Can we find a workaround for this later?

Reply:
Yeah that's fine.